net/cloudreve/Makefile

```diff
@@ -35,7 +35,7 @@ define Package/cloudreve
   SUBMENU:=Cloud Manager
   TITLE:=A project helps you build your own cloud in minutes
   URL:=https://cloudreve.org
-  DEPENDS:=@(aarch64||arm||i386||i686||x86_64) +ca-bundle
+  DEPENDS:=@(aarch64||arm||i386||i686||riscv64||x86_64) +ca-bundle
```

Comment:
Wasn’t there a GO_ARCH_DEPENDS variable or something like that?

Reply:
`mips/mips64/mips64el/mipsel/powerpc64` are unsupported by cloudreve, though i can use `@!(...)` but this line would be even longer